app/src/main/java/org/mozilla/fenix/theme/Typography.kt

```diff
@@ -0,0 +1,163 @@
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.theme
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.Typography
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+val FenixTypography = Typography(
+
+//    h1 = , Not currently in design system
+//    h2 = , Not currently in design system
+//    h3 = , Not currently in design system
+//    h4 = , Not currently in design system
+
+    /** Currently not in-use. */
+    h5 = TextStyle(
+        fontSize = 24.sp,
+        fontWeight = FontWeight.W400,
+        letterSpacing = 0.18.sp,
+        lineHeight = 24.sp,
+    ),
+
+    /** Used for headings on Onboarding Modals and App Bar Titles. */
+    h6 = TextStyle(
+        fontSize = 20.sp,
+        fontWeight = FontWeight.W500,
+        letterSpacing = 0.15.sp,
+        lineHeight = 24.sp,
+    ),
+
+    /** Used for Lists. */
+    subtitle1 = TextStyle(
+        fontSize = 16.sp,
+        fontWeight = FontWeight.W400,
+        letterSpacing = 0.15.sp,
+        lineHeight = 24.sp,
+    ),
+
+    /** Currently not in-use. */
+    subtitle2 = TextStyle(
+        fontSize = 14.sp,
+        fontWeight = FontWeight.W500,
+        letterSpacing = 0.1.sp,
+        lineHeight = 24.sp,
+    ),
+
+    /** Currently not in-use. */
+    body1 = TextStyle(
+        fontSize = 16.sp,
+        fontWeight = FontWeight.W400,
+        letterSpacing = 0.5.sp,
+        lineHeight = 24.sp,
+    ),
+
+    /** Used for body text. */
+    body2 = TextStyle(
+        fontSize = 14.sp,
+        fontWeight = FontWeight.W400,
+        letterSpacing = 0.25.sp,
+        lineHeight = 20.sp,
+    ),
+
+    /** Used for Buttons. */
+    button = TextStyle(
+        fontSize = 14.sp,
+        fontWeight = FontWeight.W500,
+        letterSpacing = 0.25.sp,
+        lineHeight = 14.sp,
+    ),
+
+    /** Used for captions. */
+    caption = TextStyle(
+        fontSize = 12.sp,
+        fontWeight = FontWeight.W400,
+        letterSpacing = 0.4.sp,
+        lineHeight = 16.sp,
+    ),
+
+    /** Used for Sheets. */
+    overline = TextStyle(
+        fontSize = 10.sp,
+        fontWeight = FontWeight.W400,
+        letterSpacing = 1.5.sp,
+        lineHeight = 16.sp,
+    ),
+)
+
+/** Used for headings on Cards, Dialogs, Banners, and Homepage. */
+val Typography.h7: TextStyle
```

Comment:
Wondering why h7 and h8 are the only ones that appears outside of FenixTypography

Reply:
The Typography class does not have them as part of their in-built system, so I had to make them extensions

Reply:
I tried just making a custom `Typography` class to bake it all together, but the `Typography`  class is `final`, unfortunately.